pipeline-steps/terraform-plan.yaml

```diff
@@ -89,6 +89,20 @@ steps:
           Write-Host "##vso[task.setvariable variable=SUBSCRIPTION_ID;isOutput=true]$subscriptionId"
           Write-Host "Subscription ID: $subscriptionId"
 
+  - ${{ if and( eq(parameters.activity_name, 'Video_Hearing_Core'), eq(parameters.environment, 'stg') ) }}:      
+    - task: AzureCLI@2
```

Comment:
Can this be swapped to use the template please
https://github.com/hmcts/azure-devops-templates/blob/master/templates/Azure/Common/get-service-connection-details.yaml

Reply:
removed.